msbuild/Xamarin.iOS.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs

```diff
@@ -236,6 +246,84 @@ bool Compile (PDictionary plist)
 			return !Log.HasLoggedErrors;
 		}
 
+		void SetRequiredArchitectures (PDictionary plist)
+		{
+			PObject capabilities;
+
+			if (plist.TryGetValue (ManifestKeys.UIRequiredDeviceCapabilities, out capabilities)) {
+				if (capabilities is PArray) {
+					var architectureValues = new HashSet<string> (new[] { "armv6", "armv7", "arm64" });
```

Comment:
We don't support `armv6` anymore (we haven't for years), you can remove all related code.

Reply:
@jstedfast you didn't remove `armv6` here, I think it should be so it can be later removed at L265.

Reply:
The "armv6" value here is just a key to remove from the Info.plist, not a value to set.